styleguide/components/61-layouts/general/overview-layout/_overview-layout.scss

```diff
@@ -43,9 +43,11 @@
   }
 
   > ul {
-    display: flex;
-    flex: 1 1 100%;
-    flex-wrap: wrap;
+    @include tablet {
```

Comment:
Why is it no longer flex below tablet?
Wont the teasers get too large?

Reply:
Fixed.